src/script/components/TitleBar/TitleBar.tsx

```diff
@@ -144,6 +144,12 @@ export const TitleBar: React.FC<TitleBarProps> = ({
   const {setCurrentView: setView} = useAppMainState(state => state.responsiveView);
 
   const setLeftSidebar = () => setView(ViewType.LEFT_SIDEBAR);
+  const {close: closeRightSidebar} = useAppMainState(state => state.rightSidebar);
+
+  const onClickBack = () => {
```

Comment:
As a general rule of thumb, functions that you declare should have an imperative name to explain what it's doing. 

The rationale for this is that you are implementing the function that you will pass as a callback to some event. But at this point the `event` is not really relevant to the function. What is interesting is explaining what the function is actually doing:

```js
onClick={onClick} => this doesn't carry any information
onClick={closePanel} => this line carries the information of what is going to happen when a click is done
```